lib/pymedphys/tests/experimental/pinnacle/test_pinnacle_cli.py

```diff
@@ -45,10 +45,10 @@
 from pathlib import Path
 from zipfile import ZipFile
 
-import pydicom
-
 from pymedphys._imports import pytest
 
+import pydicom
```

Comment:
No worries, done. Remind me to pick your brain about these imports tomorrow.

Reply:
> Remind me to pick your brain about these imports tomorrow.

Yup, before then though I recommend doing a little test to see how they work. Uninstall pydicom, then within a python shell run:

```
from pymedphys._imports import pydicom, numpy

np.linspace(0, 1)  # do something with numpy
sequence = pydicom.Sequence()  # do something with pydicom
```

And you'll see that it no longer errors at import time (can still do the numpy things), it only errors at "use time". Essentially it's an implementation of lazy imports. The import only occurs on first use. Designed so that pymedphys can be installed with minimal or no dependencies if so desired.